pages/pipelines/build_matrix.md.erb

```diff
@@ -0,0 +1,126 @@
+# Build Matrix
+
+Build Matrices lets you simplify complex build configurations by expanding a step template and array of matrix elements into multiple jobs.
+
+{:toc}
+
+For example, instead of writing three separate jobs for builds on **macOS**, **Linux** and **Windows**, like the following build configuration (which does not use a build matrix):
+
+```yaml
+steps:
+  - label: "macOS build"
+    command: "echo '.buildkite/steps/build-binary.sh macOS'"
+  - label: "Linux build"
+    command: "echo '.buildkite/steps/build-binary.sh Linux'"
+  - label: "Windows build"
+    command: "echo '.buildkite/steps/build-binary.ps1 Windows'"
+```
+{: codeblock-file="pipeline.yml"}
+
+You can use a Build Matrix, to expand a single step into three steps by replacing the `{{matrix}}` variable in the label and [command step](/docs/pipelines/command-step) with each value provided in the `matrix` array. You can interpolate the matrix values into command steps (including step labels) and into [environment variables](/docs/pipelines/environment-variables).
```

Comment:
> to expand a single step into three steps by replacing the `{{matrix}}` variable in the label and **command step**

I think it's in the label and command _attributes_ of a command step, rather than in the label and _command step_.

(Also not sure about the comma after “You can use a Build Matrix”)